packages/core/src/client.ts

```diff
@@ -2967,6 +2977,105 @@ export class MedplumClient extends EventTarget {
     this.basicAuth = encodeBase64(clientId + ':' + clientSecret);
   }
 
+  /**
+   * Subscribes to a specified topic, listening for a list of specified events.
+   *
+   * Once you have the `SubscriptionRequest` returned from this method, you can call `fhircastConnect(subscriptionRequest)` to connect to the subscription stream.
+   *
+   * @category FHIRcast
+   * @param topic The topic to publish to. Usually a UUID.
+   * @param events An array of event names to listen for.
+   * @returns A `Promise` that resolves once the request completes, or rejects if it fails.
+   */
+  async fhircastSubscribe(topic: string, events: FhircastEventName[]): Promise<SubscriptionRequest> {
+    if (!(typeof topic === 'string' && topic !== '')) {
+      throw new TypeError('Invalid topic provided. Topic must be a valid string!');
+    }
+    if (!(typeof events === 'object' && Array.isArray(events) && events.length > 0)) {
+      throw new TypeError(
+        'Invalid events provided! Events must be an array of event names containing at least one event!'
+      );
+    }
+
+    const subRequest = {
+      channelType: 'websocket',
+      mode: 'subscribe',
+      topic,
+      events,
+    } satisfies PendingSubscriptionRequest as PendingSubscriptionRequest;
```

Comment:
That's correct, but we kind of what that behavior here. The `satisfies` checks if it conforms to the type exactly, and the `as` overrides it to `PendingSubscriptionRequest` erases the "const" assertion, or something like that. I added the `as` because it was needed for one type error due to the request being overly narrow and `satisfies` is just checking it exactly conforms (as doesn't have that guarantee, you can have extra fields).

Reply:
I think that means you can remove the `satisfies` then?